src/models/StyledComponent.js

```diff
@@ -181,14 +181,17 @@ export default (ComponentStyle: Function, constructWithOptions: Function) => {
       attrs,
     } = options
 
+    const styledComponentId = (options.displayName && options.componentId) ?
+      `${options.displayName}-${options.componentId}` : componentId
```

Comment:
It defaults to an autogenerated one :) `componentId = generateId(options.displayName),`

@griffinmichl can you put in a rule that doesn't change the `styledComponentId` to this concatenated version if `options.displayName` is present, but `componentId` is not, since it'll default to sth that is similar to the `displayName`?